docs/tutorials/02_neural_network_classifier_and_regressor.ipynb

```diff
@@ -152,51 +154,80 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 43,
    "id": "specified-youth",
    "metadata": {},
    "outputs": [],
    "source": [
+    "# callback function that draws a live plot when the .fit() method is called\n",
+    "def callback(x, y):\n",
+    "    clear_output(wait=True)\n",
+    "    objective_func_vals.append(y)\n",
+    "    plt.title(\"Objective function value against iteration for NeuralNetworkClassifier\")\n",
+    "    plt.xlabel(\"Iteration\")\n",
+    "    plt.ylabel(\"Objective function value\")\n",
+    "    plt.plot(range(len(objective_func_vals)), objective_func_vals)\n",
+    "    plt.show()\n",
+    "\n",
```

Comment:
I'd rather add a text cell explaining what callback is and how it can be used. Other than that looks nice. I hope, the callback function can be reused across different models.

Reply:
I've added a text cell explaining the callback function just before a code cell only containing the callback function. Let me know if the content in the text cell is alright or if it should be changed.